src/taipy/core/data/abstract_sql.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         parent_ids: Optional[Set[str]] = None,
         last_edit_date: Optional[datetime] = None,
         job_ids: List[JobId] = None,
+        version: str = "latest",
```

Comment:
@jrobinAV On the current version, we are having a problem that when the data node, or the graph is changed, the user needs to delete `./.data` folder (on FS repo) to have the correct result, right?

So by setting the default `version` to `"latest"` here,  my idea is to make the development mode as the default mode, meaning that when the graph is changed, all old entities are deleted to prevent the above problem.
What is your thought on this?

Reply:
Right. 

I fully agree with the mechanism, and the "latest" naming is also good for me. However, I prefer to encapsulate this logic inside the `VersionManager`. It could decide what the version value should be if no value is provided. In our case it will be "latest". It s just a matter of code design, not functionality.
We can imagine in the future making the `VersionManager` smarter and choosing another value depending on some context information.

Reply:
OK, great.